### PR TITLE
container: install NetworkManager-team plugin

### PR DESCRIPTION
**Pull request by ffmancera**
Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

File: packaging/Dockerfile.centos8-nmstate-dev [hunk 1]
```diff
@@ -17,6 +17,7 @@ RUN dnf copr enable nmstate/ovs-el8 -y && \
 RUN dnf -y install --setopt=install_weak_deps=False \
                    NetworkManager \
                    NetworkManager-ovs \
+		   NetworkManager-team \
```
Comment (EdDev) [inline packaging/Dockerfile.centos8-nmstate-dev]: Please fix the indentation.

File: packaging/Dockerfile.fedora-nmstate-dev [hunk 2]
```diff
@@ -11,6 +11,7 @@ RUN bash ./docker_enable_systemd.sh && rm ./docker_enable_systemd.sh
 RUN dnf -y install --setopt=install_weak_deps=False \
                    NetworkManager \
                    NetworkManager-ovs \
+		   NetworkManager-team \
```
Comment (EdDev) [inline packaging/Dockerfile.fedora-nmstate-dev]: Please fix the indentation.

**Comment by tyll** [comment]
If the plugin is needed now, is there also a proper error message in case it is missing? Also it should be added as `Suggests` to the spec I guess.

**Comment by ffmancera** [comment]
> If the plugin is needed now, is there also a proper error message in case it is missing? Also it should be added as `Suggests` to the spec I guess.

Yes, I am adding that in the implementation PR. This is just to update the container (needed for CI in the implementation PR)

File: packaging/Dockerfile.centos8-nmstate-dev (hunk 1 above)
Comment (ffmancera) [reply packaging/Dockerfile.centos8-nmstate-dev]: Oops! Done!

File: packaging/Dockerfile.fedora-nmstate-dev (hunk 2 above)
Comment (ffmancera) [reply packaging/Dockerfile.fedora-nmstate-dev]: Oops! Done!

**Comment by tyll** [comment]
> > If the plugin is needed now, is there also a proper error message in case it is missing? Also it should be added as `Suggests` to the spec I guess.
> 
> Yes, I am adding that in the implementation PR. This is just to update the container (needed for CI in the implementation PR)

If you change the container specification (Dockerfiles) in the implementation PR, the CI should rebuild the container images before running the tests. Therefore this change should not require a separate PR. Am I missing something?

**Comment by ffmancera** [comment]
> > If the plugin is needed now, is there also a proper error message in case it is missing? Also it should be added as `Suggests` to the spec I guess.
> 
> Yes, I am adding that in the implementation PR. This is just to update the container (needed for CI in the implementation PR)

AFAIK, CI builds the container from the docker.io image, so when the PR triggers the CI, it will fetch the container from docker.io.

**Comment by tyll** [comment]
> AFAIK, CI builds the container from the docker.io image, so when the PR triggers the CI, it will fetch the container from docker.io.

Please take a look at https://github.com/nmstate/nmstate/blob/master/automation/run-tests.sh#L195 it will rebuild the image locally if necessary.

**Comment by ffmancera** [comment]
This is not needed. Done in #658

**Pull request closed without merging**